lib/ai.ts

```
@@ -0,0 +1,244 @@
+      : "Other";
+    return finalCategory;
+  } catch (error) {
+    console.error("❌ Error categorizing expense:", error);
+    return "Other";
+  }
+}
+
+export async function generateAIAnswer(
+  question: string,
+  context: ExpenseRecord[],
+  insightId?: string
```

ESLint

[error] 'insightId' is defined but never used.

(@typescript-eslint/no-unused-vars)